### ISSUE TITLE
How to handle parent

### ISSUE THREAD

**Issue by hemphillda** [GravityLabGeo/stacjs#6]
In stac.js, a tree is constructed dynamically as requests to catalogs and items are made. However, it's possible to have multiple tree variations constructed from the same application based on how the order navigation takes place (if there is more than one path to the same node).

Furthermore, in the STAC spec, one of the "rel" tags defines "parent". Nether stac.js nor stacb.html recognizes the rel tag. Instead it uses tree navigation to track a "back" capability. 

If a client directly accesses a link catalog (skipping the root), stac.js implicitly treats the link catalog as the root (since that is the entry point). However, it's possible and valid that navigation to the defined parent (as defined by the link catalog's rel=parent) be available.

Basically, there are two methods for defining parent (dynamic discovery and as defined). Both methods together are likely to be incompatible at times. Relying on the parent rel tag being accurately defined places the burden on the catalog creator to ensure correctness. If using absolute referencing, moving the catalog en masse, becomes difficult (even to make a copy of it).
